generator/lib/src/class_element_ex.dart

```diff
@@ -149,7 +152,8 @@ extension ClassElementEx on ClassElement {
 
       final objectType = ObjectType.values[modelInfo.value.getField('type')!.getField('index')!.toIntValue()!];
 
-      final mappedFields = fields.realmInfo.toList();
+      // Realm Core requires computed properties at the end so we sort them at generation time versus doing it at runtime every time.
+      final mappedFields = fields.realmInfo.toList()..sort((a, b) => a.isComputed ^ b.isComputed ? (a.isComputed ? 1 : -1) : -1);
```

Comment:
I'm still not sure this sorting is super readable. How about:

```suggestion
      final mappedFields = fields.realmInfo.toList()..sort((a, b) => a.isComputed == b.isComputed ? 0 : (a.isComputed ? 1 : -1));
```

Or even simpler:

```suggestion
      final mappedFields = fields.realmInfo.toList()..sort((a, b) => a.isComputed ? 1 : -1);
```

Reply:
Not a dealbreaker though - it should still be correct, just feels a little complicated to understand.

Reply:
I like to read it like: If a and b are different, then if a is computed, a goes last, otherwise a goes first.